core-bundle/src/Resources/public/core.js

```diff
@@ -1329,6 +1329,34 @@ var Backend =
 		});
 	},
 
+	/**
+	 * Copy the interactive help button image
+	 *
+	 * @author Ralf Baussenwein
+	 */
+	copyInteractiveHelpFromButtonsImage: function (buttons) {
+		buttons && buttons.each(function(button) {
+			var img = button.getElement('img');
+			img && button.set('title', img.get('alt'));
+		});
+	},
+
+	/**
+	 * Hide the interactive help
+	 *
+	 * @author Ralf Baussenwein
```

Comment:
please don't add @author, we have a Git commit log for that 😊

Reply:
okay okay, I thought this is the code style in the js file, because there are lots of copyrights 😁